src/main.ts

```diff
@@ -34,23 +33,17 @@ async function getWorkspaceId(client: AxiosInstance) {
 async function getChangeSet(client: AxiosInstance, workspaceId: string) {
   const changeSetsUrl = `/api/public/v0/workspaces/${workspaceId}/change-sets`
 
-  let changeSetId = core.getInput('changeSetId')
```

Comment:
I like the removal of changeSetId, viewId and managementPrototypeId from the yaml; they are useful as escape hatches, but having them front and center just makes people wonder if they *have* to specify them.

I'm less keen on removing the *ability* to tell the action to run in an existing changeSetId, however, at least not for the stated reasons: it's already 100% optional, if you don't specify the changeSetId parameter a change set will be created for you ("create" bring the default value).

Reply:
Asking people to know a changeSet Id from a CI/CD pipeline means they need to go to the UI and find it before they run the pipeline - that changeSetId may not even exist when they are running the action - it's better to just fire a new changeSet and then we can allow them to either merge or see it

Trying to find existing changeSets is going to be a frustrating experience - if we show people this is the way then this will become the way